src/main/external-resources/UMS.conf

```diff
@@ -596,6 +596,15 @@ disable_transcode_for_extensions =
 # troubleshooting when you are not able to edit the renderer configuration.
 # Default: ""
 force_transcode_for_extensions =
+
+# Disable all transcoding
+# -----------------------
+# Do not transcode under any circumstances.
+# Warning: This will probably mean that your renderer will fail to play many
+# files or fail to show subtitles, but there are cases where this is still 
```

Comment:
Telling that it's for testing and debugging purpose should be more appriopriate, pertinent, imo.
Better to strongly advertise the users, just in case...

That's said, i don't see the point to keep 2 similar test/debug functions: "disable_subtitles" and "disable_transcode_for_extensions"